lib/charms/opensearch/v0/constants_charm.py

```diff
@@ -108,7 +108,7 @@
 KibanaserverRole = "kibana_server"
 
 # Opensearch Snap revision
-OPENSEARCH_SNAP_REVISION = 51  # Keep in sync with `workload_version` file
```

Comment:
Revert this change, please

Reply:
Sure, this was not intended to go through, just to confirm the tests are passing with 2.13.0.

Reply:
@reneradoi the goal is to merge with 2.13.0 or 2.14.0 pinned?